solace-message-router/README.md

```diff
@@ -62,21 +69,24 @@ You need to fill in the following fields (* marks the mandatory ones):
 | **BASICS**                 |  |
 | Subscription*              | Provide your subscription to use. |
 | Resource Group*            | A new group, or an existing group that will be available from the pull-down menu once "Use existing" is selected. |
-| Location*                  | Specify a location for the resource group where metadata about the resources is stored. Select the region most suitable to you. |
 | **SETTINGS**               |  |
+| Region*                  | Specify an Azure region for the deployment. Note that not every resource is available in every region. |
```

Comment:
I don't see region...

Reply:
`Region` seems to be added automatically to the template, along with `Subscription` and `Resource Group`. Visible on the Azure console.